pkg/steps/multi_stage.go

```diff
@@ -353,8 +353,10 @@ func (s *multiStageTestStep) generatePods(steps []api.LiteralTestStep, env []cor
 		if link, ok := step.FromImageTag(); ok {
 			image = fmt.Sprintf("%s:%s", api.PipelineImageStream, link)
 		} else {
-			stream, _ := s.config.ImageStreamFor(image)
-			image = fmt.Sprintf("%s:%s", stream, image)
+			dep := api.StepDependency{Name: image}
```

Comment:
Sorry, implicit in that statement was that you also need to expose those dependencies to the graph builder - you'll need to do this resolution and use `LinkForImage()`, expose using the step's `Requires()` method